test/checkpointctl.bats

```diff
@@ -246,7 +246,7 @@ function teardown() {
 	checkpointctl inspect "$TEST_TMP_DIR2"/test.tar --stats
 	[ "$status" -eq 0 ]
 	[[ ${lines[8]} == *"CRIU dump statistics"* ]]
-	[[ ${lines[12]} == *"Memwrite Time"* ]]
+	[[ ${lines[12]} == *"Memwrite time"* ]]
```

Comment:
Could you create a new pull request for this patch (fix: capitalisation in tree output)?